src/ui_framework/README.md

```diff
@@ -1,4 +1,4 @@
-# Kibana UI Framework
+  # Kibana UI Framework
```

Comment:
I guess my fingers twitched and hit the wrong keys while I was trying to keep down my Starbucks.